### DISCUSSION TITLE
Template for Joomla! 4

### DISCUSSION THREAD

**Discussion by Magnytu2** [twbs/bootstrap#32669] [Q&A]
Hello everyone, sorry to intervene here. I have just launched a very important challenge. I want to create a template for the community only under bootstrap 5 for Joomla! 4. With a very complete admin.
I am looking for help for example in the format of a help forum. I have already started but am having issues building and calling bootstrap 5 build.
I would like to create your blog view first, but with the menu navigation which is in the documentation. If I approach, the sub menus do not work.
I am French, but not a developer. My Github page is: https://github.com/Magnytu2/Marie-Louise
Thanks for your help.

**Comment by rinka88** [comment]
Hello, you can participate in [this discussion](https://github.com/joomla/joomla-cms/discussions/31765)

**Reply by Magnytu2** [reply]
Hello and thanks for the link. I know these long and never-ending exchanges. I do not wish to wait for a choice of Joomla OSM on this subject. We have been waiting for Joomla for 5 years! 4. I have been trying all versions for five years. The 4 backend template versions, the three 3 frontend template versions. I thank all the developers, all the people who work on a voluntary basis. Thanks thanks thanks ...
But I think we shouldn't wait any longer and switch to Bootstrap 5. That we need a complete template management that is accessible to new Joomla users. With pure bootstrap to stick to the perfect bootstrap 5.0 documentation. I wrote a post on the French forum, no response. I posted another item on the Joomla.org forum: https://forum.joomla.org/viewtopic.php?f=803&t=983867
Joomla! 4 is complete and will be released in 2021. I'm optimistic. So I want Joomla to also have a complete, simple, clean, easy, manageable template for everyone.
